packages/components/src/ResourceList/Resource/Resource.scss

```diff
@@ -86,8 +92,19 @@
 	}
 
 	:global {
+		h3 {
+			margin-top: 0;
+			white-space: nowrap;
+			text-overflow: ellipsis;
+			overflow: hidden;
+		}
 		dl {
-			padding: 0 $padding-smaller;
+			padding-left: $padding-small;
+			margin-bottom: 0;
+
+			div:first-child {
```

Comment:
why do you need to target only the first div on the children?

Reply:
i want to target the case number 5 in frontify